Manifold/Expression.swift

```diff
@@ -13,14 +13,55 @@ public enum Expression: Hashable, IntegerLiteralConvertible {
 		self = Abstraction(x, Box(body))
 	}
 
+	public init(constant: ConstantExpression) {
+		self = Constant(constant)
+	}
+
+
+	public enum ConstantExpression: Hashable {
+		case Unit
+		case Bool(Swift.Bool)
```

Comment:
`ConstantExpression`s are a convenient way to inject specific known types into the term graph for e.g. testing type inference. They’re useful in actual programming, too!